{
  "comments": [
    {
      "key": {
        "uuid": "9ab8dda7_bb383aa1",
        "filename": "functest/opnfv_tests/features/sdnvpn.py",
        "patchSetId": 1
      },
      "lineNbr": 18,
      "author": {
        "id": 5441
      },
      "writtenOn": "2017-04-11T02:31:55Z",
      "side": 1,
      "message": "How about updating \"dir_sfc_functest\" to one name more related with sdnvpn. Otherwise, it may look confused with sfc.",
      "range": {
        "startLine": 18,
        "startChar": 8,
        "endLine": 18,
        "endChar": 24
      },
      "revId": "682a062895ce527338a66769c2e5c91662579b71",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    }
  ]
}